commission_formula/views/commission_view.xml

```diff
@@ -29,13 +17,13 @@
                     <div attrs="{'invisible':[('commission_type','!=','formula')]}">
                         <h2 name="hr_instructions">Instructions</h2>
                         <span class='instructions'>
-  Use 'line' to access all the line fields.
-  As field names may vary, you should differentiate the
-  formulas for sale.order.line and account.move.line.
-  You can, for instance, use line._name == 'sale.order.line'
-  or line._name == 'account.move.line'.
+                            Use 'line' to access all the line fields.
```

Comment:
Why this indentation?

Reply:
I did it for more readable code. But I didn't expect that will effect the UI. I reverted the changes.